solutions/usaco-920.mdx

```diff
@@ -5,10 +5,63 @@ title: The Great Revegetation
 author: Sofia Yang
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_revegetate_silver_feb19.html)
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_revegetate_silver_feb19.html)
 
 <LanguageSection>
 
+<CPPSection>
+```cpp
+// Source: Official Analysis (http://www.usaco.org/current/data/sol_revegetate_silver_feb19.html)
+#include <iostream>
+#include <fstream>
+#include <vector>
+using namespace std;
+ 
+const int MAX_N=1e5;
+int L[MAX_N+1];
+vector<int> S_nbrs[MAX_N+1],   D_nbrs[MAX_N+1];
```

Comment:
```suggestion
vector<int> S_nbrs[MAX_N+1], D_nbrs[MAX_N+1];
```
also address this:  https://github.com/cpinitiative/usaco-guide/pull/2165/files#r702349783